src/store/tests/test-summary/test-summary.actions.ts

```diff
@@ -31,7 +31,7 @@ export const IdentificationUsedChanged = createAction(
 
 export const IndependentDrivingTypeChanged = createAction(
   '[Test Summary] Independent driving changed',
-  (independentDriving: IndependentDriving) => ({ independentDriving }),
+  (drivingType: IndependentDriving) => ({ drivingType }),
```

Comment:
Does this need to change? Purposely changed the variable name to align with the key in the reducer

Reply:
agree, restored to independentDriving to match all other references to it.  Reducer/pagestate etc